src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php

```diff
@@ -105,7 +104,6 @@ protected function createConfigurationDefinition()
         $annotations = new FixerOptionBuilder('annotations', 'List of annotations to remove, e.g. `["author"]`.');
         $annotations = $annotations
             ->setAllowedTypes(array('array'))
-            ->setDefault(array())
```

Comment:
sure not a bug? configuring this as `array()` will result the fixer doing nothing